src/scripts/plugin.js

```diff
@@ -51,11 +68,23 @@ const editScriptAreaHTML = (language = 'python3') => {
   `;
 };
 
+const changeAllLanguages = (editor, language = 'python3') => {
+  // changes the data-language attribute of all pre tags
+  const blockList = editor.document.getElementsByTag('pre');
```

Comment:
Same issue here. This will change all `pre` tags even if it is not a thebelab instance.

Reply:
Addressed in 2bde441